src/components/BackgroundMedia/__tests__/BackgroundMedia.test.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {qaIdByDefault as qaIdOfBackgroundImage} from '../../../components/BackgroundImage/BackgroundImage';
+import {qaIdOfAnimatedDiv} from '../../../components/Media/Image/Image';
+import {qaIdByDefault as qaIdOfMedia} from '../../../components/Media/Media';
+import {qaIdOfSourceByDefault, qaIdOfVideoByDefault} from '../../../components/Media/Video/Video';
+import BackgroundMedia from '../BackgroundMedia';
+
+const qaId = 'background-media-component';
+
+const imageUrl =
+    'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/img-gray.png';
+const videoProps = {
+    src: [
+        'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/header-bg-video_light.mp4',
+    ],
+};
+
+describe('BackgroundMedia', () => {
+    test('render component by default', async () => {
+        render(<BackgroundMedia qa={qaId} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toBeInTheDocument();
+    });
+
+    test('add className', () => {
+        const className = 'my-class';
+
+        render(<BackgroundMedia qa={qaId} className={className} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveClass(className);
+    });
+
+    test('render with animated', async () => {
```

Comment:
Don't we have a common test for the animation?

Reply:
common  test for the animation is added